src/base/macros/model_wrapper.jl

```diff
@@ -1,6 +1,6 @@
 
 """
-    _inherit_model_methods_impl(mtype::Symbol, arglist, access, fwdlist, fns...)
+$(TYPEDSIGNATURES)
 
 A helper backend for [`@_inherit_model_methods`](@ref) and
 [`@_inherit_model_methods_fn`](@ref).
```

Comment:
Q: the macros lower in this file can't be documented with DSE?

(Also, would it be possible to use DSE again for the generated docstring below?)